Caltech-IPAC/firefly
# DM-14867, IRSA-2466: fixed bugs in tickets

DM-14867, IRSA-2466: fixed bugs in tickets

    - IRSA-2466: Only HiPS or AllSky FITS will recenter when table tabs switch
    - DM-14867: Toolbar button and then "Select new Image" buttons is active when image plot fails in API
    - API UI Test framework modified to allows for resizing the IFrames

_To Test:_

DM-14867: 

- load http://localhost:8080/firefly/test/tests-main.html
- look at test #4.  It should have a failed plot. 
- using the toolbar button you should be about to replace the plot

IRSA-2466:

- http://localhost:8080/firefly
- load 1 or more fits images at m31
- load 1 or more HiPS at m31
- goto catalogs and load two catalogs. 100 arcsec,  2mass and wise at m31
- scroll the FITS and HIPS away from the current center
- switch table table tabs
- The HiPS should recenter the FITS (non-allsky) should not.

Testing Frame work:

- Notice that each test can be resized now.
